client/derive_config.ts

```diff
@@ -6,20 +6,21 @@ import { createCache } from "./create_cache.ts";
 /** Derives host and endpoint. */
 function deriveHostEndpoint(
   region: string,
-  port: number,
+  port?: number,
+  host?: string
```

Comment:
can u pls change these to `port = 8000` and `host = "localhost"`
and drop the defaults in the func body